README.md

```diff
@@ -90,6 +90,35 @@ Using this flow is less secure since the token is long-lived. You can provide th
 2. Setting the environment variable `STACKIT_SERVICE_ACCOUNT_TOKEN`
 3. Setting it in the credentials file (see above)
 
+# Backend configuration
+
+To keep track of your terraform state, you can configure an [s3 backend](https://developer.hashicorp.com/terraform/language/settings/backends/s3) using [STACKIT Object Storage](https://docs.stackit.cloud/stackit/en/object-storage-s3-compatible-71009778.html).
+
+To do so, you need an Object Storage [s3 bucket](https://docs.stackit.cloud/stackit/en/basic-concept-objectstorage-71009785.html#BasicConceptObjectStorage-Buckets) and [credentials](https://docs.stackit.cloud/stackit/en/basic-concept-objectstorage-71009785.html#BasicConceptObjectStorage-Credentials) to access it. If you need to create them, check [Getting Started - Object Storage](https://docs.stackit.cloud/stackit/en/getting-started-objectstorage-71009792.html).
+
+Once you have everything setup, you can configure the backend by adding the following block to your terraform configuration:
+
+```
+terraform {
+  backend "s3" {
+    bucket = "bucket_name"
+    key    = "path/to/key"
+    endpoints = {
+      s3 = "https://object.storage.eu01.onstackit.cloud"
+    }
+    region                      = "eu01"
+    skip_credentials_validation = true
+    skip_region_validation      = true
+    skip_s3_checksum            = true
+    skip_requesting_account_id  = true
+    secret_key                  = "secret_key"
+    access_key                  = "access_key"

```

Comment:
There fields are actual credentials specific to each bucket right? 
If yes, maybe replace by 

```suggestion
    secret_key                  = "SECRET_KEY"
    access_key                  = "ACCESS_KEY"
```

Reply:
Same for other fields which need to be adapted by the user